Mutex/functions/Unlock-Mutex.ps1

```diff
@@ -19,24 +22,45 @@
         PS C:\> Get-Mutex | Release-Mutex
 
         Release the lock on all mutexes managed.
+
+    .EXAMPLE
+        PS C:\> Get-Mutex | Release-Mutex -Force
+
+        Release the lock on all mutexes managed regardless how often they have been locked in the process.
+
+    .NOTES
+        If using -Force the mutex will be released until a
+        MethodInvocationException is thrown. This happens if the mutex has
+        been already released.
     #>
     [CmdletBinding()]
     Param (
         [Parameter(Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
         [string]
-        $Name
+        $Name,
+        [Parameter(Mandatory = $false, ValueFromPipeline = $false, ValueFromPipelineByPropertyName = $false)]
```

Comment:
This `[Parameter()]` attribute is redundant, as all these values default to $false anyway.
Including them means you actually need to start reading the details in the param block, to understand how each parameter behaves, rather than being able to skim this.
Would you mind dropping them?

Reply:
Dropped, next push will include it.